great_expectations/datasource/fluent/spark_azure_blob_storage_datasource.py

```diff
@@ -40,17 +34,30 @@
 except ImportError:
     pass
 
+_MISSING: Final = object()
+
+if TYPE_CHECKING:
+    from great_expectations.datasource.fluent.spark_file_path_datasource import (
+        CSVAsset,
+    )
+
 
 class SparkAzureBlobStorageDatasourceError(SparkDatasourceError):
     pass
 
 
+@public_api
 class SparkAzureBlobStorageDatasource(_SparkFilePathDatasource):
+    # class attributes
+    data_connector_type: ClassVar[
+        Type[AzureBlobStorageDataConnector]
+    ] = AzureBlobStorageDataConnector
+
     # instance attributes
     type: Literal["spark_abs"] = "spark_abs"
 
     # Azure Blob Storage specific attributes
-    azure_options: Dict[str, Any] = {}
+    azure_options: Dict[str, Union[ConfigStr, Any]] = {}
```

Comment:
I missed this early. This allows the keys in `azure_options` to be substituted via config values.